docs/pages/blog/mui-core-v5.md

```diff
@@ -0,0 +1,560 @@
+> However [browser support](https://caniuse.com/flexbox-gap) for the flexbox `gap` CSS property is lacking in Safari.
+
+You can find [more details](/components/stack/) in the documentation.
+
+### Promotion from the lab
+
+We have promoted six component from the lab, after over two years iterating on feedback:
+
+- [Autocomplete](/components/autocomplete/)
+- [Pagination](/components/pagination/)
+- [Rating](/components/rating/)
+- [Skeleton](/components/skeleton/)
+- [Speed Dial](/components/speed-dial/)
+- [Toggle Buttons](/components/toggle-button/)
+
+### New in the lab
+
+The lab hosts the incubator components that are not yet ready to move to the core.
+The main difference between the lab and the core is how the components are versioned.
+Having a separate lab package allows us to release breaking changes when necessary while the core package follows a [slower-moving policy](/versions/#release-frequency).
+
+The following components are now available in the lab:
+
+- [LoadingButton](/components/buttons/#loading-buttons). It does what you would expect. It renders the `Button` in a loading/pending state.
+- [TrapFocus](/components/trap-focus/). This component traps the focus within a DOM node. For instance, it's used by the Modal for accessibility reasons.
+- [Masonry](/components/masonry/). One great use case for this component is when using the `Grid` component leads to wasting space. It's frequently used in dashboards.
+
+  <a href="/components/masonry/"><img src="/static/blog/mui-core-v5/masonry.png" alt="" style="width: 505px; margin-bottom: 16px;" /></a>
+
+### Date pickers
+
+- Transfer of ownership from third-party to mui-org.
+
+### Data Grid
+
+- See next section
+- Covered a bit in https://material-ui.com/blog/2020-q3-update/
+
+## v4 migration
+
+- The high-level changes required
+- Installation
+- ⚓️ We have introduced a new release line: v4.x.x-deprecations.x. This release line is kept in sync with the latest version of v4 and includes actionable deprecations to ease the migration to v5.
+- The codemod, covered a bit in https://material-ui.com/blog/2021-q2-update/
+- The migration guide
+
+### Change on the supported platforms
+
+This breaking change is the opportunity to drop the support of legacy upstream dependencies.
+
+- We have updated the minimum supported TypeScript version from 3.2 to **3.5**.
+  It aims to match the policy of [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped), with versions of TypeScript that are less than two years old.
+- We have updated the minimum supported node.js version from 8.0 to **12.17**.
+  It aims to match the [LTS versions that are in maintenance](https://github.com/nodejs/Release#release-schedule) mode.
+- We have updated the minimum supported React version from 16.8 to **17.0**.
+  The breaking changes released between the two versions are [minors](https://reactjs.org/blog/2020/10/20/react-v17.html).
```

Comment:
```suggestion
  The breaking changes released between the two versions are [very limited](https://reactjs.org/blog/2020/10/20/react-v17.html).
```

To avoid confusion between "minor = small scope" and "minor = minor semver release"